### DISCUSSION TITLE
SEQ and moving to HTTPS

### DISCUSSION THREAD

**Discussion by bikatla** [datalust/seq-tickets#1373] [Help]
I am trying to change SEQ from accepting logs from serilog using HTTPS instead of HTTP and default port number (5341).  
I have set the thumbprint for the private locally generated certificate.
I have set the api.ListenUris to https://localhost
I have restarted the server. 

I now can use https://<ipaddress> to browse the logs (which i used to do by http://<ipaddress>:5341

I have changed log setup in my program that uses serilog to log til https://<ipaddress>

There are however no logging to seq occurring.   When I monitor with  Fiddler I see that the response from <ipaddress>:443 is 200. 

Unfortunately my license is "old" : 5.0. 2817 

Server is a VM in azure and firewall is open for port 443 both inside windows and on the NSG in Azure.

**Comment by KodrAus** [comment]
Hi @bikatla :wave:

Would you be happy to reach out to us on our `support@datalust.co` address with a diagnostic report from your server? You can find this by visiting _Settings > Diagnostics > Download a full diagnostic report_.
